lib/languageclient.ts

```diff
@@ -421,8 +421,8 @@ export class LanguageClientConnection extends EventEmitter {
    * @param params The {CodeLens} identifying the code lens to be resolved with full detail.
    * @returns A {Promise} containing the {CodeLens} fully resolved.
    */
-  public codeLensResolve(params: lsp.CodeLens): Promise<lsp.CodeLens | null> {
-    return this._sendRequest("codeLens/resolve", params)
+  public codeLensResolve(params: lsp.CodeLens): Promise<lsp.CodeLens> {
+    return this._sendRequest(lsp.CodeLensResolveRequest.type, params)
```

Comment:
Would this be a breaking change since we are changing a public API?

Reply:
The LanguageClient class has not been exported from main.ts, but it may still be a breaking change. I do not know well which.😰

Reply:
Maybe just a note in the changelogs